demo/playground/Dockerfile.acapy

```diff
@@ -0,0 +1,20 @@
+FROM ghcr.io/hyperledger/aries-cloudagent-python:py3.9-indy-1.16.0-0.8.1
```

Comment:
Not sure about this so no worries if not changed, but wondered if you can use `../../docker/Dockerfile.demo` somehow.